mifosng-android/src/main/java/com/mifos/mifosxdroid/dialogfragments/syncsurveysdialog/SyncSurveysDialogFragment.java

```diff
@@ -206,7 +206,8 @@ public void showNetworkIsNotAvailable() {
     @Override
     public void showSurveysSyncSuccessfully() {
         btnCancel.setVisibility(View.INVISIBLE);
-        btnHide.setText(getResources().getString(R.string.dialog_action_ok));
+        dismissDialog();
+        Toast.makeText(getActivity(), "Sucessfully Synced", Toast.LENGTH_SHORT).show();
```

Comment:
Use dynamic string, please :)

Reply:
Ok

Reply:
@robustTechie, please don't use hardcoded text, can you please update your pull request? (^_^)

Reply:
Done

Reply:
> @robustTechie, please don't use hardcoded text, can you please update your pull request? (^_^)

Done